lib/scoring/util/solveCubic.js

```diff
@@ -4,55 +4,62 @@ const Big = require('bignumber.js');
 /**
  * Calculate the cubic root of the given number
  *
- * @param {number|BigNumber} x The value
+ * @param {number} x The value
```

Comment:
It receives Numbers and outputs Numbers. Only inside the function BigNumbers are used.